src/components/AddRemoveFavourite.tsx

```diff
@@ -1,5 +1,7 @@
 import React, { FC } from 'react';
-import styled from '../../../styling/styled';
+import styled from '../styling/styled';
+import FullHeart from '../assets/icons/heart_full.png';
```

Comment:
I'd rather stay with pascal case only for components and use camel case instead.

Reply:
sure, changed that